templates/emails/group.expense.approved.for.host.hbs

```diff
@@ -0,0 +1,40 @@
+Subject: New expense approved on {{group.name}}: {{{currency expense.amount currency=expense.currency}}} for {{expense.title}} by {{user.username}}
+
+{{> header}}
+
+<style>
+.btn {
+  color: #fff;
+  background: #80d823;
+  border: 1px solid transparent;
+  border-radius: 28px;
+  display: inline-block;
+  padding: .75rem 1rem;
+  text-transform: uppercase;
+  text-align: center;
+  font-weight: 700;
+  min-width: 150px;
+  box-sizing: border-box;
+  vertical-align: middle;
+}
+
+.btn.red {
+  background: #dd2663;
+}
+</style>
+
+<center>
+  <h1>{{{currency expense.amount currency=expense.currency}}}</h1>
+  <div>{{expense.title}}</div>
+  <p><font color="#999">{{expense.notes}}</font></p>
```

Comment:
Would be good to show it again who filed the expense originally and who approved it.